tests/phpunit/class-wc-stripe-webhook-handler-agentic-test.php

```
@@ -281,122 +272,91 @@ public function test_process_checkout_session_completed_skips_when_payment_inten
-				'checkout/sessions/cs_456?expand[]=payment_intent.agent_details&expand[]=line_items',
-			],
-			'session id with special chars' => [
-				'cs_test/special&chars',
-				[],
-				'checkout/sessions/cs_test%2Fspecial%26chars?expand[]=payment_intent.agent_details',
-			],
-		];
+		$this->assertNotNull( $captured_opts );
+		$this->assertArrayHasKey( 'stripe_version', $captured_opts );
+		$this->assertEquals( WC_Stripe_API::AGENTIC_COMMERCE_API_VERSION, $captured_opts['stripe_version'] );
 	}
```

PHPMD (2.15.0)

[error] The method test_api_version_override_applied is not named in camelCase. (undefined)

(CamelCaseMethodName)